common/app/model/Formats.scala

```diff
@@ -198,9 +198,31 @@ object ContentTypeFormat {
   implicit val genericThriftAtomFormat = GenericThriftAtomFormat
   implicit val recipeThriftAtomFormat = RecipeThriftAtomFormat
   implicit val reviewThriftAtomFormat = ReviewThriftAtomFormat
+  implicit val storyquestionsThriftAtomFormat = StoryquestionsThriftAtomFormat
   implicit val recipeAtomFormat = Json.format[RecipeAtom]
   implicit val reviewAtomFormat = Json.format[ReviewAtom]
-  implicit val atomsFormat = Json.format[Atoms]
+  implicit val storyquestionsAtomFormat = Json.format[StoryQuestionsAtom]
+
+
+  implicit val atomsWrite = Json.writes[Atoms]
+
+  /* Everytime you add a new atom above you will need to add a line for that atom below.
+   *
+   * We default to an Empty array if we can't find the field in the pressed fronts json. This prevents
+   * json serialisation errors at runtime and means that fronts do not have to be repressed simply for adding a new
+   * atom here when the fronts will never need to use them.
+   *
+   * */
+  implicit val reads: Reads[Atoms] = (
+    (__ \ "quizzes").read[Seq[Quiz]].orElse(Reads.pure(Nil)) and
+      (__ \ "media").read[Seq[MediaAtom]].orElse(Reads.pure(Nil)) and
+      (__ \ "interactives").read[Seq[InteractiveAtom]].orElse(Reads.pure(Nil)) and
+      (__ \ "recipes").read[Seq[RecipeAtom]].orElse(Reads.pure(Nil)) and
+      (__ \ "reviews").read[Seq[ReviewAtom]].orElse(Reads.pure(Nil)) and
+      (__ \ "storyquestions").read[Seq[StoryQuestionsAtom]].orElse(Reads.pure(Nil))
+    )(Atoms.apply _)
+
+
```

Comment:
Agreed. In term of data, None would mean the field is not present, Seq.empty means field is present but doesn't contain any atom. (I give you that from the app perspective this is the same)
I don't think models should necessary match how they are represented in json. I am just throwing an idea (and its associated drawbacks) that you summarise well: Having to define and maintain our own `Reads` vs handling optional fields when working with `Atoms`.

Thanks for your work anyway. It looks like we can improve the way we support Atoms in general in the frontend codebase.